src/test/java/PlaylistTests.java

```diff
@@ -9,8 +9,37 @@ public class PlaylistTests extends BaseTest {
     public void createPlaylist() throws InterruptedException {
```

Comment:
Please delete everywhere `throws InterruptedException`

Use Implicit and Explicit Wait in Selenium WebDriver